store/searchlayer/layer_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package searchlayer_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/mattermost/mattermost-server/v5/store/searchlayer"
+
```

Comment:
Nit: there is no need for gaps here and on L16

Reply:
damn this is my editor doing `goimports` on save.